src/llama-kv-cache-recurrent.h

```diff
@@ -15,13 +15,18 @@
 //       see the implementation of llama_kv_cache_unified_state_i for an example how to do it
 class llama_kv_cache_recurrent : public llama_memory_i {
 public:
+
+    // this callback is used to filter out layers that should not be included in the cache
```

Comment:
This is copy-pasta from the `unified` cache. I had originally hoisted it to the `llama_kv_cache` intermediate base class, but since that's been removed, I figured it was better to just copy it rather than put it into the top-level interface class.